apps/web-giddh/src/app/subscription/subscription.component.html

```diff
@@ -1,353 +1,382 @@
-                            >
-                                {{ element?.renewalDate }}
+        <button
+            class="buy-plan-btn position-absolute"
+            mat-stroked-button
+            color="primary"
+            [routerLink]="['/pages/user-details/subscription/buy-plan/' + subscriptionId]"
+            *ngIf="activeTabIndex === 3 && selectedCompany?.planVersion === 1 && !isSubscriptionLoading"
+        >
+            {{ commonLocaleData?.app_buy_new_plan }}
+        </button>
+        <mat-tab-group
+            mat-stretch-tabs="false"
+            mat-align-tabs="start"
+            animationDuration="0ms"
+            [selectedIndex]="activeTabIndex"
+            (selectedTabChange)="selectTab($event)"
+        >
+            <mat-tab [label]="localeData?.auth_key?.tab_heading">
+                <div class="wrap-tab-content-pd-15">
+                    <div class="box clearfix mt-3">
+                        <form name="authKeyForm" #authKeyForm="ngForm" novalidate role="form">
+                            <div class="form-group row align-items-center" [ngClass]="{ 'mb-1': isCopied }">
+                                <div class="col-12 col-sm-12 col-md-12 col-lg-6">
+                                    <input-field
+                                        [label]="localeData?.auth_key?.main_auth_key"
+                                        #authKeyUrl
+                                        [id]="'authKey'"
+                                        [name]="'userAuthKey'"
+                                        [(ngModel)]="userAuthKey"
+                                        [readonly]="true"
+                                    ></input-field>
+                                </div>
+                                <button type="button" mat-stroked-button color="primary" (click)="toggleIsCopied()">
+                                    {{ commonLocaleData?.app_copy }}
+                                </button>
+                            </div>
+                            <div class="row mb-3" *ngIf="isCopied">
+                                <div class="col-12 col-sm-12 col-md-12 col-lg-6">
+                                    <span class="success">{{ localeData?.auth_key?.copied }}</span>
+                                </div>
+                            </div>
+                            <div class="form-group clearfix mb-0">
+                                <button
+                                    mat-stroked-button
+                                    color="primary"
+                                    type="submit"
+                                    (click)="regenerateKey()"
+                                    class="mr-4"
+                                >
+                                    {{ localeData?.auth_key?.regenerate_auth_key }}
+                                </button>
+                                <a class="text-primary" href="{{ apiPostmanDocUrl }}" target="_blank">{{
```

Comment:
```suggestion
                                <a class="text-primary" [href]="apiPostmanDocUrl" target="_blank">{{
```

Reply:
do not change anything on old subscription module